tests/test_plotting.py

```diff
@@ -317,7 +317,7 @@ def debug_orders(pl, point):
         pl.iren._mouse_left_button_release(width, height)
         pl.iren._mouse_move(int(width * point[0]), int(height * point[1]))
 
-    mapdl.ksel("S", "node", "", 1)
+    mapdl.ksel("S", "KP", "", 1)
```

Comment:
That's a fairly significant find. Really would prefer MAPDL to issue errors rather than allowing for commands to silently fail.

Reply:
Indeed, Good idea to have PyMAPDL raises those "ignored" messages.